openc3-cosmos-cmd-tlm-api/app/controllers/tables_controller.rb

```diff
@@ -1,4 +1,4 @@
-# encoding: ascii-8bit
+# encoding: utf-8
```

Comment:
Changed because of the weird character in line 200. SimpleCov complained with `/opt/homebrew/lib/ruby/gems/3.2.0/gems/simplecov-0.22.0/lib/simplecov/source_file.rb:202:in readlines': "\xEF" from ASCII-8BIT to UTF-8 (Encoding::UndefinedConversionError)`